packages/asset-swapper/src/utils/market_operation_utils/fills.ts

```diff
@@ -71,6 +71,24 @@ function hasLiquidity(fills: Fill[]): boolean {
     return true;
 }
 
+export function ethToOutputAmount({
```

Comment:
Not sure if this is the best place to put this logic but I didn't want to create yet another file for this general util 🙈